itest/re-org_test.go

```diff
@@ -85,14 +85,15 @@ func testReOrgMint(t *harnessTest) {
 
 	// Let's wait until we see that the proof for the first asset was
 	// updated to the new block height.
-	waitForProofUpdate(t.t, t.tapd, assetList[0], newBlockHeight)
+	WaitForProofUpdate(t.t, t.tapd, assetList[0], newBlockHeight)
 
 	// We now try to validate the issuance proof of the two assets we
 	// minted again. The re-org watcher should have updated the proofs and
 	// pushed them to the proof store. They should be valid now.
 	for idx := range assetList {
 		a := assetList[idx]
-		assertAssetProofs(t.t, t.tapd, a)
+		chainClient := t.tapd.cfg.LndNode.RPC.ChainKit
```

Comment:
non-blocking nit: Here and in other loops where we use `chainClient`, we reassign the variable inside the loop instead of before the loop. That seems unnecessary?

Other loops at lines 224, 350, etc.

Reply:
good catch, your version makes the code more readable too, so added :+1: